query-node/mappings/src/content/video.ts

```diff
@@ -123,12 +134,42 @@ export async function processCreateVideoMessage(
     reactionsCount: 0,
   })
 
-  if (metadata) {
-    await processVideoMetadata(ctx, video, metadata, newDataObjectIds)
+  if (metadata && 'rawAction' in metadata) {
```

Comment:
The only bulletproof way to differentiate between the 2 types is now the `appId` (`rawAction` is optional in `AppAction`):
```suggestion
  if (metadata && 'appId' in metadata) {
```